athlinks-ontology.ttl

```diff
@@ -6,542 +6,692 @@
-                        rdfs:label "Duathlon" .
-:Swim                   a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in a Swimming athletic event" ;
-                        rdfs:label "Swim" .
-:MountainBike           a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in a Mountain Bike athletic event" ;
-                        rdfs:label "Mountain Bike" .
-:Cycling                a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in a Cycling athletic event" ;
-                        rdfs:label "Cycling" .
-:Snow                   a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in a Snowshow Running athletic event" ;
-                        rdfs:label "Snowshow Running" .
-:Adventure              a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in an Adventure athletic event" ;
-                        rdfs:label "Adventure" .
-:Obstacle               a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in an Obstacle Course athletic event" ;
-                        rdfs:label "Obstacle" .
-:Other                  a owl:NamedIndividual, :ActivityType ;
-                        rdfs:comment "Instance of a flag indicating activity in an athletic event not described" ;
-                        rdfs:label "Other" .
-
-#################################################################
-#    General axioms
-#################################################################
-
-[ rdf:type owl:AllDisjointClasses ;
-  owl:members ( 
-                :RunUpTo5k
-                :Run5kTo15k
-                :Run15kToHalfMarathon
-                :RunHalfToFullMarathon
-                :RunMarathon
-                :RunUltra
-                :MultisportSprint
-                :MultisportOlympic
-                :MultisportHalfIronman
-                :MultisportIronMan
-                :Aquathlon
-                :Aquabike
-                :Duathlon
-                :Swim
-                :MountainBike
-                :Cycling
-                :Snow
-                :Adventure
-                :Obstacle
-                :Other
-              )
-] .
+	rdfs:subPropertyOf schema:position ;
+	rdfs:domain :AthleteResult ;
+	rdfs:range [
+          rdf:type owl:Class;
+          owl:unionOf (
+            xsd:integer
+            xsd:string
+          );
+        ];
+        rdfs:comment "Rank of a course time result in the context of a particular athlete cohort. The rank may differ from its [position](http://schema.org/position), which is assumed to be sequential. Two items with sequential positions can share the same rank."@en ;
```

Comment:
I don't have a question on this one...I just appreciate the wording here to distinguish between rank and position because I didn't initially understand it.